### PR TITLE
LinkedIn connect

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
#311 @championswimmer

### CODE REVIEW

**Comment by VibhorCodecianGupta** [comment]
@championswimmer please review

File: src/passport/strategies/user/strategy-linkedin.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                  })
+                   const user = await models.User.findById(oldUser.id)
+
+                   if (user) {
+                      return cb(null, user.get())
+                  } else {
+                      return cb(null, false, {message: "Could not retrieve existing Linkedin linked account"})
+                  }
+              }
+          } else {
```
Comment (championswimmer) [inline src/passport/strategies/user/strategy-linkedin.js]: I believe this entire else block need not exist at all

**Comment by VibhorCodecianGupta** [comment]
@championswimmer changed, please review

**Comment by VibhorCodecianGupta** [comment]
@championswimmer changed, please review

**Comment by VibhorCodecianGupta** [comment]
@championswimmer changed, please review

**Comment by VibhorCodecianGupta** [comment]
@championswimmer resolved conflicts, please review

**Comment by VibhorCodecianGupta** [comment]
@championswimmer any updates?

**Comment by championswimmer** [comment]
oauth changes takes time to review. need to set out time to create linkedin account with Coding Blocks credentials and test in production. possibly tomorrow

**Comment by championswimmer** [comment]
if you're unable to fix the issues before the contest ends, I can award you 1500 bounties for the work till now

**Comment by VibhorCodecianGupta** [comment]
@championswimmer fixed the requested issues, opened a fresh PR #335 along with a motley PR

**Comment by VibhorCodecianGupta** [comment]
@championswimmer what's the claimable bounty here?

**Pull request merged**